application/freshdesk/v0/get_all.go

```diff
@@ -0,0 +1,110 @@
+package freshdesk
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/instill-ai/component/base"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func (c *FreshdeskClient) GetAll(objectType string, pagination bool, paginationPath string) ([]TaskGetAllResponse, string, error) {
+
+	resp := []TaskGetAllResponse{}
+
+	httpReq := c.httpclient.R().SetResult(&resp)
+
+	var rawResp *resty.Response
+	var err error
+	if !pagination {
```

Comment:
I think we should set some limitation to avoid to fetch all data if the data count is big.
And, we probably can set it to fetch the latest 100 piece of data.
And, we need to write it down in the document